operations/addresses.md

```diff
@@ -0,0 +1,240 @@
+# Addresses
+
+## Get all addresses
+
+Returns all addresses associated with the specified accounts within the enterprise. This operation uses [Pagination](../guidelines/pagination.md)
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/addresses/getAll`
+
+```javascript
+{
+    "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+    "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+    "Client": "Sample Client 1.0.0",
+    "AccountIds": [
+        "3db2c989-7d95-42b4-a502-a9f246db1634",
+        "1f841f0a-2af3-4f01-9441-ca52828d5224",
+        "5e045eb0-8d57-4bf5-a766-1fa28e7dfd11",
+    ],
+    "AddressIds": [
+        "6cabdd7b-6c6b-4939-919a-0d043997fa7a",
+        "cfc63db6-ccae-44ad-9e58-48aa89b301ed"
+    ],
+    "Limitation": { "Count": 10 }
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `AccountIds` | array of string | required, max 1000 items | Unique identifiers of [Companies](companies.md#company) or [Customers](customers.md#customer) within the enterprise. |
+| `AddressIds` | array of string | optional, max 1000 items | Unique identifiers of [Addresses](configuration.md#address) addresses within the enterprise. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of address items returned. |
+
+### Response
+
+```javascript
+{
+    "Addresses": [
+        {
+            "Id": "3db2c989-7d95-42b4-a502-a9f246db1634",
+            "AccountId": "078892fe-93f0-46b8-9b33-af4600df2c65",
+            "AccountType": "Customer",
+            "Line1": "Address Line 1",
+            "Line2": "Address Line 2",
+            "City": "City",
+            "PostalCode": "10 000",
+            "CountryCode": "ES",
+            "CountrySubdivisionCode": "ES-BA",
```

Comment:
Can we put better example data here? I know it's JUST example data but could be better. For example form different part of documentation.
```
"Address": {
    "Id": "bab7441c-4b82-43bc-8001-ab0400a346ec",
    "Line1": "Rheinlanddamm 207-209",
    "Line2": "",
    "City": "Dortmund",
    "PostalCode": "44137"
    "CountryCode": "DE",
    "CountrySubdivisionCode": null,
    "Latitude": null,
    "Longitude": null
}
```